app/models/foreman_deployments/resource_models/create_resource.rb

```diff
@@ -0,0 +1,19 @@
+module ForemanDeployments
+  module ResourceModels
+    #
+    class CreateResource < ActiveModel::Model
+      include ActiveModel::Dirty
+      include ActiveModel::Conversion
+      extend ActiveModel::Naming
+
+      attr_accessor :resource_type, :definition_hash, :computed_hash
+
+      def new_resource
+        debugger
```

Comment:
Can you please remove the debugger? Even though it's only an early preview it can bite one who tries it.